src/App.scss

```diff
@@ -17,8 +17,8 @@
     &.ins-c-application-info {
         height: 100%;
 
-        img.ins-c-application-info__logo {
-            @include rem('height', 56px);
+        img.ins-c-application-info__logo svg {
+            @include rem('height', 70px);
```

Comment:
Abstract this. 56px was the standard that we wanted to use. If this is trying to target the modal, make a new selector for it.